entgql/schema.go

```diff
@@ -773,6 +773,13 @@ func (e *schemaGenerator) mapScalar(gqlType string, f *gen.Field, ant *Annotatio
 				case "[]string":
 					scalar = "[String!]"
 				}
+			case reflect.Map:
+				if f.Type.RType.Ident == "map[string]interface {}" {
```

Comment:
the spacing between `interface` and `{}` really weird. But the test passed

Reply:
Yeah, I was confused when I saw that at first but I think it's just how the `reflect` package identifies `interface{}`. You can see the same format in some of their tests: https://github.com/golang/go/blob/master/src/reflect/all_test.go#L713